SFDigitalServices/sf-dahlia-web
# DAH 1336 additional housing opportunities page

[DAH-1336](https://sfgovdt.jira.com/browse/DAH-1336)

Changes:
 - Added additional housing opportunity resources
 - Pull in updated InfoCardGrid from Bloom UIC
 - Added resource copy to en.json
 - Override css variables
 - Add e2e tests
 -FIx some contrast errors based on WAVE analysis


Review Steps:
- Go to /additional-resources?react=true
- Ensure everything looks okay, is responsive, accessible, etc.
- Compare to Figma (linked in ticket)

[DAH-1336]: https://sfgovdt.jira.com/browse/DAH-1336?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
